code/modules/projectiles/ammo_datums.dm

```diff
@@ -2692,6 +2692,34 @@
 	shell_speed = AMMO_SPEED_TIER_2
 	added_spit_delay = 0
 
+
+/datum/ammo/xeno/acid/marking
+	name = "marking spit"
+
+	damage= 15
+	shell_speed = AMMO_SPEED_TIER_2
+	accuracy = HIT_ACCURACY_TIER_10*8
+	flags_ammo_behavior = AMMO_SKIPS_ALIENS
+	max_range = 32
+
+/datum/ammo/xeno/acid/marking/on_hit_mob(mob/M, obj/item/projectile/P)
+	. = ..()
+	M.AddComponent(/datum/component/bonus_damage_stack, 100, world.time)
+	for(var/mob/living/carbon/Xenomorph/X in range(5, M))
+
+		var/overshield_amount = 100
```

Comment:
NOT A GA COMMENT, but IIRC the dev team doesn't want more shield abilities and is interested in actively removing the preexisting ones. This probably won't fly